src/DispatchListener.php

```diff
@@ -216,7 +216,7 @@ protected function marshalBadControllerEvent(
         $controllerName,
         MvcEvent $event,
         Application $application,
-        \Exception $exception
+        \Throwable $exception
```

Comment:
it is not gonna work on php 5.6 :(
please have a look in [`composer.json`](https://github.com/zendframework/zend-mvc/blob/master/composer.json#L16), and it still must work on: `"php": "^5.6 || ^7.0"`